.github/workflows/docker-prepublish.yml

```diff
@@ -5,51 +5,19 @@ on:
     types: [prereleased]
 
 jobs:
-  build:
-    name: Build Webapp using npm/bower
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js
-      uses: actions/setup-node@v1
-      with:
-        node-version: 10.x
-    # https://help.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows
-    - name: Cache node modules
-      uses: actions/cache@v2
-      env:
-        cache-name: cache-node-modules
-      with:
-        # npm cache files are stored in `~/.npm` on Linux/macOS
-        path: ~/.npm
-        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
-        restore-keys: |
-          ${{ runner.os }}-build-${{ env.cache-name }}-
-          ${{ runner.os }}-build-
-          ${{ runner.os }}-
-    - name: Fetch bower
-      run: npm install -g bower grunt-cli
-    - name: npm ci
-      run: npm ci --prefer-offline
-    - name: bower install
-      run: bower install --allow-root
-    - name: Run Build
-      run: buildDate=`date +%Y%m%d` && grunt build --buildnr=$buildDate
-    - name: Upload build artifact
-      uses: actions/upload-artifact@v1
-      with:
-        name: client-kundenportal-dist
-        path: dist
+  call-workflow-build:
+    uses: OpenOlitor/openolitor-client-kundenportal/.github/workflows/grunt-build.yml@fix/deps_node16
```

Comment:
it is 18, right??

Reply:
This is the link to the file. Since it's only available in the current branch, I had to add this. As soon as on prod, I will change this and create another PR. 

And it's 16 because I had the plan to go to 16 but since 18 worked out, it's better.